bin/install_dependencies

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+BASEDIR=`dirname $0`/..
+
+apt-get update
+apt-get build-dep -y python-imaging
+apt-get install -y python-virtualenv libjpeg8 libjpeg62-dev libfreetype6 libfreetype6-dev npm
```

Comment:
Maybe we should point directrly `node` here. I know, that `npm` required `node`, or more accurate will be to say, that `npm` is bundled with `node`, but for some reason (and I understand why) You're pointing, that `node` will be istalled in `README`.